Backend/DataStructures.py

```diff
@@ -0,0 +1,64 @@
+
+# TODO: Add data classes and data structures to this file
+# Ex: Tweets class. User class
+
+
+# [User class]
+class User:
```

Comment:
there were some changes to the database design in Design Document II. This should probably add a few fields to reflect that

Reply:
Then again, if you don't think that's necessary at this point, please tell me and I'll just resolve this discussion